test/index.js

```diff
@@ -200,14 +234,18 @@ describe('KeyringController', () => {
     })
   })
 
-  describe('getAppKeyAddress', () => {
-    it('returns the expected app key address', async () => {
+  describe('getAppKeyAddress', function () {
+
+    it('returns the expected app key address', async function () {
       const address = '0x01560cd3bac62cc6d7e6380600d9317363400896'
       const privateKey = '0xb8a9c05beeedb25df85f8d641538cbffedf67216048de9c678ee26260eb91952'
-      const keyring = await keyringController.addNewKeyring('Simple Key Pair', [ privateKey ])
+
+      const keyring = await keyringController.addNewKeyring('Simple Key Pair', [privateKey])
       keyring.getAppKeyAddress = sinon.spy()
+      /* eslint-disable-next-line require-atomic-updates */
```

Comment:
I'm having a hard time imagining why this line is required. It looks like `require-atomic-updates` only applies to lines with `yield` or `await` in them? Neither keyword is in the next line

Reply:
Same, and yet:
```shell
  245:7  error  Possible race condition: `keyringController.getKeyringForAccount` might be reassigned based on an outdated value of `keyringController.getKeyringForAccount`  require-atomic-updates

✖ 1 problem (1 error, 0 warnings)
```

Reply:
Hmm. Maybe something to do with the fact that `keyringController` is re-assigned each test in the `beforeEach` hook? 🤔 

I still don't understand this, but maybe this is because the rule doesn't understand how mocha works.